github/ci/prow/files/jobs/node-maintenance-operator/OWNERS

```diff
@@ -0,0 +1,6 @@
+filters:
+  ".*":
+    reviewers:
+      - yanirq
```

Comment:
Is there really no one else we can add here?

Reply:
maybe @ksimon1 ?

Reply:
Do you want to change that still? Otherwise I'd approve and we merge?

@rmohr I suppose that this OWNERS adds up to the parent OWNERS file right?

Reply:
Yes, but I prefer if there are at least two people which feel directly responsible for a specific project.

Reply:
@rmohr  I agree,   @MarSik  do we have another candidate ?

Reply:
@ksimon1 is an option, but he basically got all our CNV work dumped on his head already.

Reply:
I've added @ksimon1 to the OWNERS file.